.azure-pipelines-templates/matrix.yml

```diff
@@ -91,14 +91,14 @@ jobs:
         suffix: 'Perf'
         depends_on: ${{ parameters.static_check_job_name }}
 
-  # HTTP build (behind a toggle for now), only on NoSGX and CFT
-  # TODO: Once large messages are supported with HTTP (e.g. quotes over join request), SGX should be run
-  - template: common.yml
-    parameters:
-      target: NoSGX
-      consensus: CFT
-      env: ${{ parameters.env.NoSGX }}
-      cmake_args: '${{ parameters.build.common.cmake_args }} ${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }} ${{ parameters.build.CFT.cmake_args }} ${{ parameters.build.http.cmake_args }}'
-      ctest_filter: '${{ parameters.test.NoSGX.ctest_args }}'
-      suffix: 'HTTP'
-      depends_on: ${{ parameters.static_check_job_name }}
+  # HTTP build (behind a toggle for now), only on CFT
+  - ${{ each target in parameters.target }}:
+    - template: common.yml
+      parameters:
+        target: ${{ target }}
+        consensus: CFT
+        env: ${{ parameters.env.NoSGX }}
+        cmake_args: '${{ parameters.build.common.cmake_args }} ${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }} ${{ parameters.build.CFT.cmake_args }} ${{ parameters.build.http.cmake_args }}'
+        ctest_filter: '${{ parameters.test.NoSGX.ctest_args }}'
+        suffix: '${{ target }}_HTTP'
```

Comment:
I'll look at a way to get the suffix into at least the end to end perf tests, otherwise we're going to lose track of the JSON-RPC numbers.